sdk/core/core-amqp/src/retry.ts

```diff
@@ -67,13 +72,29 @@ export interface RetryConfig<T> {
   /**
    * @property {number} [delayInSeconds] Amount of time to wait in seconds before making the
    * next attempt. Default: 15.
+   * When `exponentialRetry` is set to `true`, this is used to compute the exponentially increasing delays between retries.
    */
   delayInSeconds?: number;
   /**
    * @property {string} connectionHost The host "<yournamespace>.servicebus.windows.net".
-   * Used to check network connectivity.
+   * Used to check network connectivity. Applicable only when performing linear retry.
    */
   connectionHost?: string;
+  /**
+   * @property {boolean} [exponentialRetry] Flag to denote if we want to perform exponential retry and not
+   * the default, which is linear.
+   */
+  exponentialRetry?: boolean;
```

Comment:
I have generally found it helpful to name properties/variables/arguments that are boolean to have a preceding verb like `is` or `has`. In this case, I would suggest `isExponential`.

@bterlson Since the `RetryOptions` passed when creating a `EventHubClient` would need this property too, any thoughts on the name?

Reply:
Agree, updated to `isExponentialRetry` for now